app/src/main/java/de/danoeh/antennapod/fragment/BaseHomeFragment.java

```diff
@@ -0,0 +1,128 @@
+package de.danoeh.antennapod.fragment;
+
+import android.content.Context;
+import android.content.SharedPreferences;
+import android.content.res.Resources;
+import android.os.Bundle;
+import android.support.design.widget.TabLayout;
+import android.support.v4.app.Fragment;
+import android.support.v4.app.FragmentManager;
+import android.support.v4.app.FragmentPagerAdapter;
+import android.support.v4.view.ViewPager;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+
+import de.danoeh.antennapod.R;
+import de.danoeh.antennapod.activity.MainActivity;
+
+//This fragment is the base fragment of the Home Page. The HomePageFragment and CategoriesFragment will be
+//loaded in the tabs contained in the BaseHomeFragment
+public class BaseHomeFragment extends Fragment{
```

Comment:
Why create a completely new fragment when we already had HomePageFragment? This is confusing, I think you should have kept the HomePageFragment for this page and just added a new fragment for the "featured" tab. The naming is confusing now and a bit inappropriate.

Reply:
To create tab you need a base fragment to hold whatever will stay on the page when you switch view(atennapod along with the logo in this case) and hold the tablayout. The homepagefragment and categoryfragment are loaded when you switch the tabs. 

I agree the names are confusing. Other people are currently using the same fragment for other tasks and the tasks where planned to be done before my part. I was planning on renaming the HomePageFragment in FeaturesFragment upon completions of the other tasks. But I'll do it right now.

Reply:
I understand how your tab functionality works, what I don't understand is why you did not use the HomePageFragment as your base fragment, and then created a new fragment for the feature tab.